### PR TITLE
DDF-4394 Localized polling sources error message

### PR DESCRIPTION
**Pull request by bellcc**
#### What does this PR do?
- Localized error polling sources message
- Fixed undefined attribute

#### Who is reviewing it? 
@leo-sakh 
@brianfelix 

#### Select relevant component teams: 
@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.
@andrewkfiedler
@djblue

#### How should this be tested?
- Do not enable local search
- Perform a search with no sources selected

#### What are the relevant tickets?
[DDF-4394](https://codice.atlassian.net/browse/DDF-4394)

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by bellcc** [comment]
build now

File: ui/packages/catalog-ui-search/src/main/webapp/js/ApplicationSetup.js [hunk 1]
```diff
@@ -83,10 +83,8 @@ let getErrorResponse = function(event, jqxhr, settings, throwError) {
     settings.type === 'GET'
   ) {
     return {
-      title: `Error Polling ${properties.i18n['Sources']}`,
-      message: `Unable to query server for list of active ${
-        properties.i18n['sources']
-      }`,
+      title: properties.i18n['sources.polling.error.title'],
```
Comment (leo-sakh) [inline ui/packages/catalog-ui-search/src/main/webapp/js/ApplicationSetup.js]: ✏️ I would suggest making the object keys more descriptive as to what type of `title` and `message` this is so it's easier to debug/refactor/find in the future

File: ui/packages/catalog-ui-search/src/main/webapp/js/ApplicationSetup.js [hunk 2]
```diff
@@ -83,10 +83,8 @@ let getErrorResponse = function(event, jqxhr, settings, throwError) {
...
+      message: properties.i18n['sources.polling.error.message']
```
Comment (brianfelix) [inline ui/packages/catalog-ui-search/src/main/webapp/js/ApplicationSetup.js]: Is there any more context we can give to the user about why we could not query the server for the list of active sources?

Comment (bellcc) [reply ui/packages/catalog-ui-search/src/main/webapp/js/ApplicationSetup.js]: Unfortunately I don't think so. The server would have to send back more info about the error to do this.

File: ui/packages/catalog-ui-search/src/main/webapp/js/ApplicationSetup.js (hunk 1 above)
Comment (bellcc) [reply ui/packages/catalog-ui-search/src/main/webapp/js/ApplicationSetup.js]: I was hoping `source.polling.error` would be descriptive enough. It's just as much information as the error message used to give

**Comment by bellcc** [comment]
build now

File: ui/packages/catalog-ui-search/src/main/webapp/js/ApplicationSetup.js (hunk 1 above)
Comment (Lambeaux) [reply ui/packages/catalog-ui-search/src/main/webapp/js/ApplicationSetup.js]: This seems descriptive enough to me. @leo-sakh did you have a concrete example of what you were looking for?

**Comment by bellcc** [comment]
build now

**Comment by bellcc** [comment]
Full Build Locally on a fresh m2

```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 02:58 h
[INFO] Finished at: 2018-12-12T02:03:18-07:00
[INFO] ------------------------------------------------------------------------
```

**Comment by mojogitoverhere** [comment]
hero in progress

**Comment by bellcc** [comment]
build now

**Comment by mojogitoverhere** [comment]
## 🎉 Hero build was successful! 🎉 ##

1. Verified `All Sources` is present in the Sources dropdown
2. Disabled the local catalog and ran a search. Verified the error message was `No sources are currently selected. Edit the search and select at least one source. `
3. Shutdown DDF and waited for the sources to be polled. Verified the error title was `Error Polling Sources` and the message was `Unable to query server for list of active sources`.

**Pull request merged**